ovos_audio/service.py

```diff
@@ -253,6 +255,29 @@ def run(self):
         else:
             self.status.set_error('No TTS loaded')
 
+    def handle_b64_audio(self, message):
+        """syntehsizes speech, but instead of queuing for playback
+        returns it b64 encoded in the bus
+        allows 3rd party integrations to use OVOS as a TTS service
+        """
+        sess = SessionManager.get(message)
+        stopwatch = Stopwatch()
+        stopwatch.start()
+        utterance = message.data['utterance']
```

Comment:
Do you want it to throw an exception if the message isn't formatted properly?

Reply:
it wil throw a KeyError in that case

i guess we could introduce a exception for MalformedBusMessage or something like that, perhaps in ovos-bus-client? adopting this pattern should then be done across the board, not only in this repo

Reply:
Might not be a bad idea, if you don't want to go so far as to enforce it with something like Pydantic. Just validating that you intend for an exception to possibly be thrown here without being caught - not sure if it would bubble up to the logs without an except statement

Reply:
the KeyError should bubble up, but i like the idea of adding a dedicated exception for this across the board :+1: